packages/blade/src/components/Popover/__tests__/Popover.web.test.tsx

```diff
@@ -0,0 +1,281 @@
+/* eslint-disable @typescript-eslint/restrict-plus-operands */
+/* eslint-disable @typescript-eslint/require-await */
+/* eslint-disable @typescript-eslint/no-empty-function */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
```

Comment:
> We should disable ESLint rules for files only when there are multiple instances of the same rule otherwise we should disable it for specific lines only.

Moved. 

> Similarly, we should aim to fix the @typescript-eslint/explicit-function-return-type rule errors, by adding the return types:

I prefer inferring the types than defining primitives types explicitly. Makes the code cleaner.

Reply:
> I prefer inferring the types than defining primitives types explicitly. Makes the code cleaner.

Lint/formatting rules exist so that we follow consistent practices without any personal preferences, however, in this case, we may have to re-look our ESLint rules https://github.com/razorpay/blade/issues/1169